src/main/kotlin/moody/blackjack/ui/BlackjackController.kt

```diff
@@ -0,0 +1,62 @@
+package moody.blackjack.ui
+
+import moody.blackjack.domain.card.Deck
+import moody.blackjack.domain.card.DeckShuffleGenerator
+import moody.blackjack.domain.game.BlackjackGame
+import moody.blackjack.domain.user.Dealer
+import moody.blackjack.domain.user.Player
+import moody.blackjack.domain.user.User
+import moody.blackjack.domain.user.Users
+import moody.blackjack.ui.view.InputView
+import moody.blackjack.ui.view.OutputView
+
+class BlackjackController {
+    private val blackjackGame: BlackjackGame
+
+    init {
+        val deck = Deck(DeckShuffleGenerator())
+        val names = InputView.inputPlayerNames()
+        val users = Users((names.map { Player(it) }.plus(Dealer())))
+        blackjackGame = BlackjackGame(deck, users)
+    }
+
+    fun startGame() {
+        blackjackGame.initUserCards()
+        OutputView.printUsersCards(blackjackGame.users)
+    }
+
+    fun playPlayerRound() {
+        while (blackjackGame.hasRemainingUsers()) {
+            val currentPlayer = blackjackGame.currentPlayer()
+            giveCardToPlayer(currentPlayer)
+            when {
+                currentPlayer.isBust() -> {
+                    OutputView.printUserCards(currentPlayer)
+                    OutputView.printPlayerBustMessage()
+                }
+                else -> OutputView.printUserCards(currentPlayer)
+            }
```

Comment:
```suggestion
            if (currentPlayer.isBust()) {
                OutputView.printUserCards(currentPlayer)
                OutputView.printPlayerBustMessage()
                continue
            }
            OutputView.printUserCards(currentPlayer)
```
체크하는 조건이 하나라서 if 문을 사용하는 것을 제안합니다! when 대신 if 문을 사용하는 거에 대해서 무디는 어떻게 생각하시나요? ㅎㅎ

Reply:
이 부분은 if문이 맞다고 생각해요! when이 코틀린에서 나온 분기 조건문이다 보니까 습관적으로 when을 사용해보자라는 의견에 매몰된 것 같네요ㅋㅋㅋ